lib/models/supermarkets.js

```diff
@@ -0,0 +1,12 @@
+const { getCollection } = require("../database");
+
+async function readSupermarkets() {
+  const supermarketCollection = await getCollection("supermarkets");
+  const supermarkets = await supermarketCollection.find({}).toArray();
+  if (!supermarkets) {
+    return null;
+  }
```

Comment:
I remember to ask you this question before:
In which case is `supermarkets` falsy?
I don't think this case will ever happen here.

Reply:
It should be zero, if someone would delete them from mongoDB, isn't it?

Reply:
@JonasImm 
I think it should be an empty array (`[]`). 
There is not `zero` in JavaScript.

Is an empty array falsy or truthy?